### PR TITLE
Boiler plate code

### PR DESCRIPTION
**Pull request by abelfeleke**
Starter code for trips

### CODE REVIEW

**Review by thechungster — COMMENTED** [review]
The target folder is generated code so generally a the `target/` should be in the `.gitignore` file so they don't get committed/pushed

**Comment by abelfeleke** [comment]
Made another commit to boiler about .gitignore

File: .project [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (thechungster) [inline .project:1]: For the .project, .settings, and target files you may have to revert because they were committed before the .gitignore was updated

Comment (abelfeleke) [reply .project:1]: I think I'll just scratch these commits because we are using Firebase and I think we won't be needing servlets anymore!

Comment (thechungster) [reply .project:1]: Sounds good!

**Pull request closed without merging**